internal/cmd/run.go

```diff
@@ -317,3 +344,60 @@ func confirmExecution(cmd *cobra.Command, numTasks int, parallel bool, category
 
 	return nil
 }
+
+func promptForEnvVar(cmd *cobra.Command, ev CommandExportExtractMatch) (string, error) {
+	label := fmt.Sprintf("Set Environment Variable \" %s\"", ev.Key)
+	ip := prompt.InputParams{Label: label, PlaceHolder: ev.Value}
+	if ev.HasStringValue {
+		ip.Value = ev.Value
+	}
+	model := tui.NewStandaloneInputModel(ip, tui.MinimalKeyMap, tui.DefaultStyles)
+
+	finalModel, err := newProgram(cmd, model).Run()
+	if err != nil {
+		return "", err
+	}
+	val, ok := finalModel.(tui.StandaloneInputModel).Value()
+	if !ok {
+		return "", errors.New("canceled")
+	}
+	return val, nil
+}
+
+func replaceVarValue(ev CommandExportExtractMatch, newValue string) string {
+	parts := strings.SplitN(ev.Match, "=", 2)
+	replacedText := fmt.Sprintf("%s=%s", parts[0], newValue)
```

Comment:
```suggestion
	replacedText := fmt.Sprintf("%s=%q", parts[0], newValue)
```

RHS should be quoted